components/content-types/BannerCarousel/BannerCarousel.js

```diff
@@ -0,0 +1,131 @@
+import { Component } from 'react';
```

Comment:
Why class components with old state instead of functional components with hooks?